packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -6,6 +6,10 @@ import { ISemanticTextColors } from './ISemanticTextColors';
  * The collection of all semantic slots for colors used in themes.
  *
  * Note: text colors are defined in ISemanticTextColors.ts.
+ * We are aware that the usage of semantic slots vs. palette slots is not consistent across the Fabric controls code.
+ * Both offerings need updates such as adding/removing slots. We've been actively making some of these changes.
```

Comment:
No, we never remove slots. It would break backwards compatibility.
We should not add more palette slots (besides neutralSecondaryAlt).